<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cangaroo</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWorkspace">
    <property name="title">
     <string>W&amp;orkspace</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="separator"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <widget class="QMenu" name="menu_New">
     <property name="title">
      <string>&amp;New</string>
     </property>
     <addaction name="action_Trace_View"/>
     <addaction name="actionLog_View"/>
     <addaction name="actionGraph_View"/>
    </widget>
    <addaction name="menu_New"/>
   </widget>
   <widget class="QMenu" name="menuMeasurement">
    <property name="title">
     <string>&amp;Measurement</string>
    </property>
    <addaction name="actionStart_Measurement"/>
    <addaction name="actionStop_Measurement"/>
    <addaction name="separator"/>
    <addaction name="actionSetup"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Trace">
    <property name="title">
     <string>&amp;Trace</string>
    </property>
    <addaction name="action_Clear"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Trace_to_file"/>
   </widget>
   <addaction name="menuWorkspace"/>
   <addaction name="menuMeasurement"/>
   <addaction name="menu_Trace"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Trace_View">
   <property name="text">
    <string>&amp;Trace View</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actionLog_View">
   <property name="text">
    <string>&amp;Log View</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
  </action>
  <action name="actionSetup">
   <property name="text">
    <string>Setup...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionStart_Measurement">
   <property name="text">
    <string>Start Measurement</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionStop_Measurement">
   <property name="text">
    <string>Stop Measurement</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionSave_Trace_to_file">
   <property name="text">
    <string>&amp;Save Trace to file...</string>
   </property>
  </action>
  <action name="actionGraph_View">
   <property name="text">
    <string>&amp;Graph View</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New...</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Clear">
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
